internal/support/scheduler/const.go

```diff
@@ -28,7 +28,6 @@ const (
 	VALUE                = "value"
 	UNLOCKED             = "UNLOCKED"
 	ENABLED              = "ENABLED"
-	SCHEDULER_TIMELAYOUT = "20060102T150405" /** Scheduler **/
 	TIMELAYOUT           = "20060102T150405" /** Scheduler **/
```

Comment:
Is this comment still valid?

Reply:
Yes :) that is ISO 8601 in general

Reply:
What is "scheduler" telling us because I don't understand the point of the comment. These are all scheduler constants.

Reply:
removed the comment :) . fixed